runtime/autoload/provider/pythonx.vim

```diff
@@ -4,6 +4,8 @@ if exists('s:loaded_pythonx_provider')
 endif
 
 let s:loaded_pythonx_provider = 1
+let s:min_python2_version = '2.6'
+let s:min_python3_version = '3.3'
```

Comment:
I think it might be a good idea to put these minimum versions in a more prominent place at the beginning of the file.